modules/tinymce/src/core/test/ts/browser/annotate/AnnotationPersistenceTest.ts

```diff
@@ -49,59 +50,82 @@ describe('browser.tinymce.core.annotate.AnnotationPersistenceTest', () => {
       attributes: {
         'data-test-anything': data.anything
       },
-      classes: [ ]
+      classes: [ 'test-class' ]
     })
   };
 
-  const setupSingleAnnotation = (editor: Editor) => {
+  const setupSingleSpanAnnotation = (editor: Editor) => {
     // '<p>This is the only p|ar|agraph</p>'
     editor.setContent('<p>This is the only paragraph <em>here</em></p>');
     editor.undoManager.add();
     TinySelections.setSelection(editor, [ 0, 0 ], 'This is the only p'.length, [ 0, 0 ], 'This is the only par'.length);
     annotate(editor, 'test-annotation', 'test-uid', { anything: 'one-paragraph' });
+    assertMarkings(editor, 1, 0);
     TinyAssertions.assertContentPresence(editor, {
-      '.mce-annotation': 1,
       'p:contains("This is the only paragraph here")': 1
     });
     editor.execCommand('undo');
+    assertMarkings(editor, 0, 0);
     TinyAssertions.assertContentPresence(editor, {
-      '.mce-annotation': 0,
       'p:contains("This is the only paragraph here")': 1
     });
     editor.execCommand('redo');
+    assertMarkings(editor, 1, 0);
     TinyAssertions.assertContentPresence(editor, {
-      '.mce-annotation': 1,
       'p:contains("This is the only paragraph here")': 1
     });
   };
 
-  const contentContains = (editor: Editor, pattern: string, isContained: boolean) => {
+  const setupSingleBlockAnnotation = (editor: Editor) => {
+    editor.setContent(`<pre class="language-markup" contenteditable="false">test</pre>`);
+    editor.undoManager.add();
```

Comment:
```suggestion
    editor.resetContent(`<pre class="language-markup" contenteditable="false">test</pre>`);
```
This might be better here as it'll make sure the undo state is cleared as well?